schemas/model_definition.schema.json

```
@@ -92,7 +92,17 @@
           "type": "integer"
         },
         "leaf_value": {
-          "type": "number"
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "number"
+              }
```

```suggestion
              },
              "minItems": 1
```
An empty array in the `leaf_value` would be bad.